agent/functional_tests/testdata/taskdefinitions/parallel-pull/task-definition.json

```diff
@@ -0,0 +1,78 @@
+{
+    "family": "concurrent-pull-test",
+    "containerDefinitions": [
+    {
+        "name": "busybox",
+        "image": "busybox",
```

Comment:
Can you use images in the local registry instead of images pulled from Docker Hub?  I'd just like to make sure we separate pull failures from Docker Hub due to network (or other) issues from pull failures related to the code under test.

It would also be a good idea to make sure that none of these images are cached in the Docker daemon prior to the functional tests starting.